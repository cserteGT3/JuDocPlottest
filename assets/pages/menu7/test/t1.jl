# This file was generated, do not modify it. # hide
using PlotlyJS, DelimitedFiles, Statistics, ORCA
fp = joinpath("assets", "temperature", "temps.tsv")
d, h = readdlm(abspath(fp), '\t', header = true)
sc = scatter(;x=d[:,1], y=d[:,2], mode="markers")
p = plot(sc)
savefig(p, joinpath(@__DIR__, "plotly2.html"), format="html")
savefig(p, joinpath(@__DIR__, "plotly1.png"), format="png")
savefig(p, joinpath(@__DIR__, "output", "t2.out"), format="html")
println("The average is:", mean(d[:,2]))