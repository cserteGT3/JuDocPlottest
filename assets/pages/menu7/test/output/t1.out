The average is:13.4
